samples/sample_multi_transcode/src/sample_multi_transcode.cpp

```diff
@@ -721,32 +721,35 @@ mfxStatus Launcher::QueryAdapters()
 
 void Launcher::ForceImplForSession(mfxU32 idxSession)
 {
-    //change only 8 bit of the implementation. Don't touch type of frames
-    mfxIMPL impl = m_InputParamsArray[idxSession].libType & mfxI32(~0xFF);
-
-    mfxU32 idx = GetPreferredAdapterNum(m_Adapters, m_InputParamsArray[idxSession]);
-    switch (m_Adapters.Adapters[idx].Number)
+    if (m_InputParamsArray[idxSession].libType != MFX_IMPL_SOFTWARE)
```

Comment:
Well, I'd prefer other condition to simplify code and reduce delta:
```
if (m_InputParamsArray[idxSession].libType == MFX_IMPL_SOFTWARE)
    return;
```

Reply:
Agreed. Done.